src/pca_report_library/compiler/xelatex.py

```diff
@@ -37,7 +43,7 @@
 
 def compile_tex(tex_file):
     """Run xelatex compiler twice."""
-    for x in range(1, 3):
+    for _ in range(1, 3):
```

Comment:
Why did you change this variable name from `x` to `_`?

Reply:
In trying to commit the changes for the import fixes, I got lots of linter errors primarily in customer.graphs.py. This was around missing docstrings, unused variables in iterations, inconsistency of camel and snake case, and some places where redefinition of builtins were happening (using str as var name). I tried to cleanup what I could without having to do an entire refactor, but this is just around the linter issues.

This specific case, was the linter complaining that x was an unused variable in the loop. While it needs to be rewritten and likely better approaches, my thought was that we could use python's "_" temp variable (Source: https://medium.com/python-features/naming-conventions-with-underscores-in-python-791251ac7097#:~:text=Conclusions%3A,for%20temporary%20variables) to satisfy the linter and not risk breaking something downstream by altering the iterations. At least until we have the time to refactor later and add test coverage.

Reply:
I had not seen that `_` convention for a throwaway variable before in Python, but it makes sense to me.  I would have preferred to see a separate commit with that change and a brief explanation, rather than have it tucked away within the large https://github.com/cisagov/pca-report-library/pull/9/commits/e8f86ff41afa561ac33e578f67896e9d8b3f0f13 commit.